packages/web-app-files/src/views/spaces/Projects.vue

```diff
@@ -1,5 +1,20 @@
 <template>
   <div class="oc-p-s">
+    <oc-button
+      v-if="hasCreatePermission"
+      id="new-space-menu-btn"
+      ref="createNewSpaceButton"
+      key="new-space-menu-btn-enabled"
+      :aria-label="$gettext('Create a new space')"
```

Comment:
Don't think that one is strictly necessary, but let's do an `a11y` review for Spaces once we're getting close to the end of the sprint/release